src/resolver.rs

```diff
@@ -701,12 +702,10 @@ impl AnnotationMapImpl {
 
     /// annotates the given statement (using it's `get_id()`) with the given type-name
     pub fn annotate(&mut self, s: &AstNode, annotation: StatementAnnotation) {
-        log::trace!("Annotation: {annotation:?} @ {s:?}");
```

Comment:
fyi I've previously introduced these trace statements but in hindsight I have to admit they're not as useful as I imagined (i.e. they're waaaaay too bloaty). A better approach would be a more concise representation but that's something I'll address (or not, who knows) when I have the pleasure to debug resolver bugs :^)

Reply:
soon<sup>TM</sup>